### PR TITLE
Add hint of support bundle namespaces

### PR DESCRIPTION
**Pull request by w13915984028**
Issue:  https://github.com/harvester/harvester/issues/5673

It is important to hint user to add related namespace into support-bundle-namespaces for collecting data.

### CODE REVIEW

**Review by bk201 — APPROVED** [review]
lgtm

File: docs/troubleshooting/harvester.md [hunk 1]
```diff
@@ -67,6 +67,16 @@ Users can generate a support bundle in the Harvester GUI with the following step
 - Enter a useful description for the support bundle and click `Create` to generate and download a support bundle.
     ![](/img/v1.2/troubleshooting/harvester-sb-support-modal.png)
 
+:::note
+
+When your workloads are deployed on customized namespaces and the issue is/maybe related to those workloads, make sure those namespaces are configured on [support-bundle-namespaces](../advanced/settings.md#support-bundle-namespaces). The support bundle only captures data from the configured namespaces.
```
Comment (jillian-maroket) [inline docs/troubleshooting/harvester.md:72]:
```suggestion
Whenever you encounter an issue that may be related to workloads deployed in custom namespaces, configure the [support-bundle-namespaces](../advanced/settings.md#support-bundle-namespaces) setting to include those namespaces as data sources. The support bundle only collects data from the configured namespaces.  
```

File: docs/troubleshooting/harvester.md [hunk 2]
```diff
@@ -67,6 +67,16 @@ Users can generate a support bundle in the Harvester GUI with the following step
...
+
+When you encounter the error of timeout, you can adjust the [support-bundle-timeout](../advanced/settings.md#support-bundle-timeout) and retry.
```
Comment (jillian-maroket) [inline docs/troubleshooting/harvester.md:74]:
```suggestion
For timeout errors, you can adjust the value of the [support-bundle-timeout](../advanced/settings.md#support-bundle-timeout) setting and then restart the support bundle generation process.  
```

File: docs/troubleshooting/harvester.md [hunk 3]
```diff
@@ -67,6 +67,16 @@ Users can generate a support bundle in the Harvester GUI with the following step
...
+
+When you plan to use the none-default container image, you can set the [support-bundle-image](../advanced/settings.md#support-bundle-image).
```
Comment (jillian-maroket) [inline docs/troubleshooting/harvester.md:76]:
```suggestion
If you intend to use a non-default container image, you can configure the [support-bundle-image](../advanced/settings.md#support-bundle-image) setting.  
```

**Review by jillian-maroket — COMMENTED** [review]
Review done

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**